documentation/docs/20-core-concepts/50-state-management.md

```diff
@@ -64,12 +111,56 @@ If you are using SSR, the `load` function will run on the server initially, whic
 <p>Welcome {$user.name}</p>
 ```
 
-If you have global data whose initial state is not dependent on a request (in other words, it's always the same), then you can keep storing that data globally, as long as you make sure you don't update it during the initial rendering on the server (during load or component render).
+## Component state is preserved
```

Comment:
```suggestion
## Shared state in the browser is ok

The previous sections have shown how to avoid accidentally leaking data on the server. In the browser we don't have this problem and can use shared state freely. If you have global data whose initial state is not dependent on a request (in other words, it's always the same), then you can keep storing that data globally, as long as you make sure you don't update it during the initial rendering on the server (during load or component render):

-- a good example I can't think of right now, something with onMount or if(browser){} --

If you turn your app into an SPA, some of the above restrictions are lifted. Your application is only rendered in the browser and you can use shared state inside shared `load` functions, too, since it also only runs in the browser. Lifting these restrictions should not be an argument in favor of moving to an SPA though: most apps benefit greatly from the increased SEO and robustness that SSR gives them.

## Component state is preserved
```

Reply:
https://github.com/sveltejs/kit/pull/9239/files#r1120120963

Reply:
Yeah I like that addition. I think it's worthwhile to add a similar note to this chapter then:
```suggestion
If you're not using SSR, then there's no risk of accidentally exposing one user's data to another during component rendering, so you can use shared state inside components without using the context feature.

## Component state is preserved
```

Reply:
minor tweak to avoid repetitious phrasing

```suggestion
If you're not using SSR (and can guarantee that you won't need to use SSR in future) then you can safely keep state in a shared module, without using the context API.

## Component state is preserved
```